packages/web/src/scss/components/Toast/_Toast.scss

```diff
@@ -0,0 +1,121 @@
+//    While the CSS spec speaks [1] about the interaction of `overflow` and `align-content` properties, there may be
+//    a blind spot in the spec because all major browsers (Firefox, Chrome, Safari) do not allow scrolling for
+//    `align-content: end`.
+//
+//    [1] https://drafts.csswg.org/css-align-3/#overflow-scroll-position
+//
+//    A similar Chromium bug with flex containers is tracked here:
+//
+//    https://issues.chromium.org/issues/41130651
+//
+// 3. Allow scrolling when the messages do not fit the screen. Scrolling is only possible over the toast message boxes
+//    and not over the entire container. This is because the `pointer-events` property of the container is set to `none`
+//    (4) so neither the toast container, nor the toast queue stands in the way of the user's interaction with the page
+//    content beneath it.
+//
+//    (The alternative would be to scroll the toast queue itself, but this would go against (4) by blocking user's
+//    interaction with the underlying content.)
+//
+// 4. Because the `Toast` container is present on the page all the time, we must allow interaction with the page content
+//    beneath it. The same applies to the toast queue element (7).
+//
+// 5. On devices with rounded displays (like iPhone X and later), prefer the safe area value over our spacing, if
+//    bigger. The `viewport-fit="cover"` viewport meta must be present in HTML for this to work. Zero value is used as
+//    a fallback.
+//
+// 6. EXPERIMENTAL: When the virtual keyboard is open, the toast container should be pushed up to make space for the
+//    keyboard. This is achieved by setting the bottom margin of the container to the height of the keyboard. This
+//    feature requires the following JavaScript snippet and is currently supported only in Chrome 94 on Android and
+//    later.
+//
+//    ```js
+//    if ('virtualKeyboard' in navigator) {
+//      navigator.virtualKeyboard.overlaysContent = true;
+//    }
+//    ```
+//
+//    ℹ️ Similar behavior can be achieved with the `interactive-widget=resizes-content` viewport meta, but then it would
+//    affect the entire page, not just the toast container. Again, this feature is currently supported only in Chrome
+//    108 on Android and later.
+//
+//    https://ishadeed.com/article/virtual-keyboard-api/
+//    https://developer.chrome.com/blog/viewport-resize-behavior/
+//
+// 7. The toast queue element is in place so we can reverse the visual order of child toast messages when vertical
+//    alignment is set to `bottom`. Reversing order of all child elements is only possible with flex containers.
+//
+// 8. Generate:
+//
+//    a) horizontal (x) and
+//    b) vertical (y) alignment classes for each breakpoint.
+
+@use 'sass:map';
+@use '../../tools/breakpoint';
+@use 'theme';
+
+.Toast {
+    --toast-padding-x: #{theme.$padding-x};
+    --toast-padding-y: #{theme.$padding-y};
+
+    position: fixed; // 1.
+    inset: 0; // 1.
```

Comment:
TODO: Fix jumping during scrolling in mobile browsers.

Reply:
A few notes on this.

1. Dynamic viewport changes and the the bottom stickyness are fixed with `bottom: 0; top: auto; height: 100<dvh/lvh/%>`:

   https://github.com/lmc-eu/spirit-design-system/assets/5614085/9098d079-8d95-497d-be0f-834a79bcd20f

2. I avoided `100dvh` in favor of `100lvh` so the queue is not clipped during dynamic viewport changes. This is what happens with `dvh`:

   https://github.com/lmc-eu/spirit-design-system/assets/5614085/4f52006c-6299-482b-9e07-46ca34a9fd16

3. Because of `pointer-events: none` that we need, **scrolling the queue on touch-only devices does not work**:

   https://github.com/lmc-eu/spirit-design-system/assets/5614085/fecaa1fd-d1f8-4820-b167-73a089d078cc

Reply:
> scrolling the queue on touch-only devices does not work

**TODO:** In the README, discourage developers from displaying too many toasts at once as it may cause the page to be unusable. This problem only has design solutions, not technical.